backend/src/model/entity.rs

```diff
@@ -11,70 +11,230 @@ use diesel::{Identifiable, Insertable, Queryable};
+    /// Enum value.
     pub herbaceous_or_woody: Option<HerbaceousOrWoody>,
+    /// The list of enum values.
     pub life_cycle: Option<Vec<Option<LifeCycle>>>,
-    pub growth_rate: Option<GrowthRate>,
-    pub mature_size_height: Option<String>,
-    pub mature_size_width: Option<String>,
+    /// The list of enum values.
+    pub growth_rate: Option<Vec<Option<GrowthRate>>>,
+    ///
+    pub height: Option<String>,
+    ///
+    pub width: Option<String>,
+    /// The list of enum values.
     pub fertility: Option<Vec<Option<Fertility>>>,
+    ///
     pub pollinators: Option<String>,
+    ///
     pub flower_colour: Option<String>,
+    /// Enum value.
     pub flower_type: Option<FlowerType>,
+    /// The creation date of the entry.
     pub created_at: NaiveDateTime,
+    /// The last update date of the entry.
     pub updated_at: NaiveDateTime,
+    ///
     pub has_drought_tolerance: Option<bool>,
+    ///
     pub tolerates_wind: Option<bool>,
+    /// The list of the references of the plant.
     pub plant_references: Option<Vec<Option<String>>>,
+    /// Boolean value indicating whether the plant is a tree. The initial value is to `True` if  herbaceous_or_woody (woody) and life_cycle (perennial)
     pub is_tree: Option<bool>,
+    /// Enum value. The initial value is to `light feeder` if "Nutritionally poor soil" in `environmental_tolerances` is present.
     pub nutrition_demand: Option<NutritionDemand>,
+    /// Number value between -1..6 (-1 should be printed as 00)
     pub preferable_permaculture_zone: Option<i16>,
+    /// Date value fetched from PracticalPlants page showing the last modification date of the plant.
     pub article_last_modified_at: Option<NaiveDateTime>,
+    ///
+    pub hardiness_zone: Option<String>,
+    /// The list of enum values.
+    pub light_requirement: Option<Vec<Option<LightRequirement>>>,
+    /// The list of enum values.
+    pub water_requirement: Option<Vec<Option<WaterRequirement>>>,
+    /// The list of enum values.
+    pub propagation_method: Option<Vec<Option<PropagationMethod>>>,
+    ///
+    pub alternate_name: Option<String>,
+    ///
+    pub diseases: Option<String>,
+    ///
+    pub edible: Option<bool>,
+    ///
+    pub edible_parts: Option<Vec<Option<String>>>,
+    ///
+    pub germination_temperature: Option<String>,
+    ///
+    pub introduced_into: Option<String>,
+    ///
+    pub layer: Option<String>,
+    ///
+    pub leaves: Option<String>,
+    ///
+    pub medicinal_parts: Option<String>,
+    ///
+    pub native_to: Option<String>,
+    ///
+    pub plants_for_a_future: Option<String>,
+    ///
+    pub plants_of_the_world_online_link: Option<String>,
+    ///
+    pub plants_of_the_world_online_link_synonym: Option<String>,
+    ///
+    pub pollination: Option<String>,
+    ///
+    pub propagation_transplanting: Option<String>,
+    ///
+    pub resistance: Option<String>,
+    ///
+    pub root_type: Option<String>,
```

Comment:
There are more values. It is my bad that I was assuming you will be looking through the database itself instead of the `plants.csv`, because in comparison to the external dataset files this one was used only for presentation purposes. I'll update it incrementally from now on as well!